db/migrate/20230110144730_add_index_to_historical_matches_members.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToHistoricalMatchesMembers < ActiveRecord::Migration[7.0]
+  def change
+    add_index :historical_matches, :members, if_not_exists: true
```

Comment:
Since this is an array field, we'll want to use a different indexing method via 

```ruby
    add_index :historical_matches, :members, using: "gin", if_not_exists: true
```

Reply:
Some context https://www.postgresql.org/docs/current/indexes-types.html